lib/index.ts

```diff
@@ -5,15 +5,39 @@ declare const Cypress: any
 declare const cy: any
 
 Cypress.Commands.add('percySnapshot', (name: string, options: any = {}) => {
-  const percyAgentClient = new PercyAgent({
-    clientInfo: clientInfo(),
-    environmentInfo: environmentInfo()
-  })
+  const percyAgentClient = new PercyAgent({ handleAgentCommunication: false })
+
+  // Use cy.exec(...) to check if percy agent is running. Ideally this would be
+  // done using something like cy.request(...), but that's not currently possible,
+  // for details, see: https://github.com/cypress-io/cypress/issues/3161
+  // We use 'healthcheck' script from npm binary directory.
+  const healthcheck = `\`npm bin\`/healthcheck localhost ${percyAgentClient.port} /percy/healthcheck`
```

Comment:
@djones this is now perhaps the most debatable line in this PR. Some research lead me to using the result of calling `npm bin` as the most robust way of finding the absolute directory to where the `healthcheck` script is, but it is possible there are better ways (the other option I considered was constructing the path based on `__dirname__`, but that felt sketchier).